frontend/src/modules/contributor/components/edit/work-history/contributor-work-history-edit.vue

```diff
@@ -71,14 +76,19 @@
         <lf-button type="secondary-ghost" @click="close">
           Cancel
         </lf-button>
-        <lf-button
-          type="primary"
-          :loading="sending"
-          :disabled="$v.$invalid || sending"
-          @click="updateWorkExperience()"
+        <lf-tooltip
+          :disabled="!(hasSameOrgDetails && !isEdit)"
+          content="Please enter a different Job title or Period, as there is already a work experience associated with the selected organization."
```

Comment:
There's another condition we should also add to disabled the button + show tooltip:
**If organization already exist and both job title and period are empty**. Can you also add that validation?